scripts/ci/run_perftest.sh

```diff
@@ -90,24 +125,31 @@ systemctl unmask influxdb.service
 systemctl start influxdb
 
 # Common variables used across all tests
-datestring=$(date +%s)
+datestring=${TEST_COMMIT_TIME}
 seed=$datestring
 db_name="benchmark_db"
 
+# Controls the cardinality of generated points. Cardinality will be scale_var * scale_var.
+scale_var=1000
+
+# How many queries to generate.
+queries=500
+
+# Lines to write per request during ingest
+batch=5000
+# Concurrent workers to use during ingest/query
+workers=4
```

Comment:
I backed down from my original batch size of 15000 and worker count of 20 because I didn't want to skew test results by overloading the machine with more concurrent requests than it could reasonably handle. When I ssh'd into an instance I saw it had 8 CPUs, so 4 workers felt reasonable (4 for the client-side and 4 for the server-side)